model/model_training/check_dataset_appearances.py

```diff
@@ -90,20 +142,35 @@ def iterate_over_dataset(ds):
     args = argument_parsing()
     pp = pprint.PrettyPrinter(indent=4)
 
+    overview_dct = {}
     train_datasets, val_datasets = {}, {}
     for dataset_name in args.datasets:
-        print(f"start with dataset {dataset_name}")
         train, val = get_one_dataset(None, dataset_name, mode=args.mode.value, data_path=args.cache_dir)
         train_datasets[dataset_name] = train
         if val is not None:
             val_datasets[dataset_name] = val
         matched_train = iterate_over_dataset(train)
         matched_val = iterate_over_dataset(val)
-        if len(matched_train) != 0:
-            pp.pprint(f"Found the following occurances in TRAIN {dataset_name}:")
-            pp.pprint(dict(matched_train))
-        if len(matched_val) != 0:
-            pp.pprint(f"Found the following occurances in VAL {dataset_name}:")
-            pp.pprint(dict(matched_val))
+        train_dct = {k: len(v) for k, v in matched_train.items()}
+        val_dct = {k: len(v) for k, v in matched_val.items()}
+        unified_keys = list(set(train_dct.keys()).union(set(val_dct.keys())))
+        unified_counts = {k: train_dct.get(k, 0) + val_dct.get(k, 0) for k in unified_keys}
+        import pdb
```

Comment:
NIT: Just a test script but maybe we should remove the debug statement here.

Reply:
ok, removed it! thanks for the review